tools/cldr-code/src/main/java/org/unicode/cldr/test/ConsoleCheckCLDR.java

```diff
@@ -554,14 +556,18 @@ public static void main(String[] args) throws Throwable {
 
                     if (CLDRFile.isSupplementalName(localeID)) return;
                     if (supplementalDataInfo.getDefaultContentLocales().contains(localeID)) {
-                        System.out.println("# Skipping default content locale: " + localeID);
+                        if (skipComments) {
+                            System.out.println("# Skipping default content locale: " + localeID);
```

Comment:
Does `skipComments` mean comments should be skipped? If so, should be `!skipComments`?